db/seeds.rb

```diff
@@ -36,23 +36,53 @@
 
 if Rails.env.development?
   Bulkrax::Importer.create(
-    name: "Trinity International University -  Evangelical Beacon",
+    name: "AMBS -  Biblioteca Digital Anabautista",
     admin_set_id: "admin_set/default",
     user_id: 1,
     frequency: "PT0S",
-    parser_klass: "Bulkrax::OaiDcParser",
-    limit: 10,
+    limit: 50,
+    parser_klass: "Bulkrax::OaiIaParser",
     parser_fields:
-      {"base_url"=>"http://collections.carli.illinois.edu/oai/oai.php",
+      {"base_url"=>"https://archive.org/services/oai.php",
        "metadata_prefix"=>"oai_dc",
-       "set"=>"tiu_beacon",
-       "institution_name"=>"Trinity International University Rolfing Library",
+       "set"=>"collection:bibliotecadigitalanabautista",
+       "collection_title"=>"AMBS -  Biblioteca Digital Anabautista",
+       "institution_name"=>"Anabaptist Mennonite Biblical Seminary",
        "rights_statement"=>"http://rightsstatements.org/vocab/UND/1.0/",
        "override_rights_statement"=>"1",
-       "thumbnail_url"=>"http://collections.carli.illinois.edu/utils/getthumbnail/collection/tiu_beacon/id/<%= identifier.split('/').last %>"},
+       "thumbnail_url"=>"https://archive.org/services/img/<%= identifier.split(':').last %>"},
     field_mapping: nil
   )
+
+  if Rails.env.development?
+    Bulkrax::Importer.create(
```

Comment:
Is there a way to check if the importer exists before creating it? I have many duplicates of the first importer.

Reply:
Here's a way this could be done: 

```ruby
Bulkrax::Importer.find_or_create_by(name: 'Trinity International University - Evangelical Beacon') do |importer|
  importer.admin_set_id = 'admin_set/default'
  importer.user_id = 1
  ...
end
```